docs/troubleshooting.md

```diff
@@ -3,51 +3,50 @@
 The Collector offers multiple ways to measure the health of the Collector
 as well as investigate issues.
 
+- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/zpagesextension/README.md)
+  extension, which by default is on port `55679`, can be used to check
+  receivers and exporters trace operations via `/debug/tracez`. `zpages`
+  often contains error logs that the collector does not emit.
```

Comment:
I don't think `servicez`, `pipelinez`, & `extensionz` are documented anywhere. Would this be the right place to add docs for those as well?